### ISSUE TITLE
[System test runner] --datasets flag is not working

### ISSUE THREAD

**Issue by ycombinator** [elastic/elastic-package#118]
```
$ elastic-package test system -v --datasets access
2020/09/28 19:25:21 DEBUG Enable verbose logging
Run system tests for the package
Done
```

**Comment by mtojek** [comment]
Which directory is it? Is it inside the nginx package?

**Comment by ycombinator** [comment]
Yes, it is under the `apache` package's root directory.

If you don't mind I'd like to take a look at this one. It would be convenient to get this fixed for efficiently working on #111.
